acceptance_tests/utilities/pubsub_helper.py

```diff
@@ -15,3 +22,46 @@ def publish_to_pubsub(message, project, topic, **kwargs):
 
     future.result(timeout=30)
     logger.info("Sent PubSub message", topic=topic, project=project)
+
+
+def consume_aims_new_address_messages(context):
+    subscription_path = subscriber.subscription_path(Config.AIMS_NEW_ADDRESS_PROJECT,
+                                                     Config.AIMS_NEW_ADDRESS_SUBSCRIPTION)
+    subscriber.subscribe(subscription_path, callback=functools.partial(store_aims_new_address_message, context=context))
+
+
+def store_aims_new_address_message(message, context):
+    context.aims_new_address_message = json.loads(message.data)
+    message.ack()
+
+
+def sync_consume_of_pubsub(context):
+    subscription_path = subscriber.subscription_path(Config.AIMS_NEW_ADDRESS_PROJECT,
+                                                     Config.AIMS_NEW_ADDRESS_SUBSCRIPTION)
+
+    response = subscriber.pull(subscription_path, max_messages=2, timeout=30)
+    test_helper.assertEqual(len(response.received_messages), 1)
+
+    context.aims_new_address_message = json.loads(response.received_messages[0].message.data)
+
+    ack_ids = []
+    for received_message in response.received_messages:
+        ack_ids.append(received_message.ack_id)
```

Comment:
```suggestion
    ack_ids = [message.ack_id for message in response.received_messages]
```
🐍  (my personal preference for list comprehensions)

Reply:
nice